frontend/packages/kubevirt-plugin/src/topology/components/kubevirtComponentFactory.ts

```diff
@@ -45,6 +45,7 @@ export const vmActions = (
       return action(model, contextMenuResource, {
         vmi,
         vmStatusBundle,
+        t: (t) => t, // FIXME(yaacov): this should be t: TFunction (e.g. useTranslation or i18n.t)
```

Comment:
@pcbailey help,  i tried using `i18n.t` here, but it was undefined :-( what am i missing ?